### ISSUE TITLE
BTabs issues

### ISSUE THREAD

**Issue by dwgray** [bootstrap-vue-next/bootstrap-vue-next#2383]
### Describe the bug

This is a list of the issues I found while porting the docs:

- [x] Pills examples isn't working - it looks like `card-header-tabs` is incompatible with `nav-pills` - may be  the same as #2047 
- [x] Keyboard nav isn't fully implmented
- [x] `underline` isn't implemented - [new BS5 feature](https://getbootstrap.com/docs/5.3/components/navs-tabs/#underline), which is implemented in `BNav` but not in `BTab`
- [x] `no-key-nav` isn't implemented
- [x] `BTab` implements `unmountOnce` - I think this is something that we intend to cover with `unmountLazy` - Should we use `useShowHide` on `BTab`?
- [x] `BTab` doesn't implement the `click` event and there is a stub `click` event in `BTabs` that should probably be remvoed
- [ ] `BTabs` doesn't implement the `changed` event 

### Reproduction

docs

### Used Package Manager

pnpm

**Comment by dwgray** [comment]
> Oh man, it's so awesome the you go through these!

Glad to help. I could probably fix some of these myself, but I think I'm more useful right now acting as a second set of eyes by going through the docs/parity report in a systematic way to try to catch these kinds of details. I think we'll have a significantly more consistent/polished product that way.

**Comment by xvaara** [comment]
@dwgray what keyboard navigation is missing? I added up&down and noKeyNav.

**Comment by dwgray** [comment]
> @dwgray what keyboard navigation is missing? I added up&down and noKeyNav.

Here's the link to BSV's description: https://bootstrap-vue.org/docs/components/tabs#keyboard-navigation

I don't know that we need to blindly copy that, but I'd like to document the difference if we don't. 

I think the main thing we're missing from a functionality perspective is some form of first and last. It looks like there is an attempt at an implementation with -999 and +999, but they don't seem to do anything in the simple example I tried (sorry - probably would have been helpful to have a little more detail in my initial report):

```vue
<template>
  <BCard no-body>
    <BTabs card>
      <BTab title="Tab 1" active>
        <BCardText>Tab contents 1</BCardText>
      </BTab>
      <BTab title="Tab 2">
        <BCardText>Tab contents 2</BCardText>
      </BTab>
      <BTab title="Tab 3">
        <BCardText>Tab contents 2</BCardText>
      </BTab>
    </BTabs>
  </BCard>
</template>
```

**Comment by xvaara** [comment]
I think I did all of those navs now in the pr linked.

**Comment by dwgray** [comment]
I think using the aria guide is great (looks like you just pushed that as I was writing this :-))

**Comment by dwgray** [comment]
Looks great. To be clear: `home` and `end` weren't working for my test case - did this PR do anything to fix that, or are those already working for you?

**Comment by xvaara** [comment]
Yes, they work. Mac chrome + safari tested.

**Comment by dwgray** [comment]
> Yes, they work in that pr, didn't work before. Only pageUp & pageDown. Mac chrome + safari tested.

Sounds good. I don't think it's important enough to pull down the PR locally to test, but I'll check on Windows when it gets to main.

**Comment by dword-design-honestly** [comment]
Same issue here.
